server/core/import_resolver.py

```diff
@@ -91,7 +95,9 @@ def _resolve_packages(file_symbol, level, from_stmt):
     return file_tree
 
 def _get_or_create_symbol(ls, symbol, names, file_symbol, asname, start_pos, end_pos):
-    """try to return sub symbol that is a file or package, or create the symbol"""
+    """try to return sub symbol that is a file or package, or create the symbol. The second returned element is the last
+    successfully loaded symbol in the search."""
+    last_symbol = symbol
```

Comment:
Why are you doing that? Is this assignment used somewhere?

Reply:
in the return statement :) It is used to know until which point the search has been, so we can identify which kind of import it is